tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/jdbc/RelationalDataSource.java

```diff
@@ -18,7 +18,7 @@
 import java.util.Iterator;
 import java.util.List;
 
-public interface JdbcTableDataSource
+public interface RelationalDataSource
```

Comment:
My idea here was to use "Relational" as a name here since data is produced as tuples for relation (row by row). Our `HiveDataSource` loads data in binary format (e.g. reading from resources and uploading to HDFS). I believe `HiveDataSource` should be renamed to `BinaryDataSource` % a few changes to the interface.
It's a work for a separate PR though.